### ISSUE TITLE
Overwrite page titel via model

### ISSUE THREAD

**Issue by jonasfeige** [fabianmichael/kirby-meta#37]
Hey,
really great plugin!

Is it possible to overwrite the page's `<title>` via its model? If I read the docs correctly, I can only add additional metadata but not overwrite anything, is that correct?

**Comment by fabianmichael** [comment]
@jonasfeige No, that’s currently not possible for the title field. What exactly do you want to do with that?

**Comment by jonasfeige** [comment]
I have a client who offers different classes, and each of the corresponding dates has a unique page. The template has auto-generated, randomized titles, because they are only used internally. Ideally, I would be able to overwrite the title and e.g. use the title of the page's parent.

**Comment by fabianmichael** [comment]
Okay, so do you think it’s necessary to override the whole title or just the part before the separator? Providing one from the model sounds like a good idea, it can still be overridden by hand if necessary.

**Comment by jonasfeige** [comment]
For my use case the page title (before the separator) would be enough. And yeah, customizability via the model with even more control via the panel would be amazing!

**Comment by fabianmichael** [comment]
@jonasfeige I’m happy to add this feature since it might be interesting to many other users. Unfortunately I don’t know when I will have the time for doing that.

**Comment by ImaCrea** [comment]
Hello here, I also would love to be able to customize title of the page.

Usecase: the website is an agenda of different events. Each event has its format, which could be 'talk', 'meeting', etc. The idea would be to append in the title of the event, the format. So a page with title 'The evolution of humans' would be 'TALK: The evolution of humans'. It would help in term of accessibility.

[edit] for the moment the 'dirty' way I found is to edit meta>snippets>general.php directly in the plugin code. 😬

**Comment by fabianmichael** [comment]
@ImaCrea I am currently working on a new version of the plugin, but don’t have a final timeline yet At the moment, sticking to your “dirty” solution is probably the best option.

**Comment by fabianmichael** [comment]
The plugin now finally supports overrides for almost anything via `$page->metaOverrides()` method (see develop branch)
